src/requestLib.js

```diff
@@ -1,44 +1,47 @@
 var http = require('http');
+var https = require('https');
 var constants = require('../config/constants');
 var Utils = require('./utils');
-var keepAliveAgent = new http.Agent({
-  keepAlive: true
-});
+var url = require('url');
+var HttpProxyAgent = require('https-proxy-agent');
```

Comment:
```suggestion
var HttpProxyAgent = require('http-proxy-agent');
```
shouldn't that be `http`?

Reply:
can be resolved